CAN-426, 427, 430, 431

vikasranatalentelgia opened the pull request:
Manage Files: Frontend design to delete files in list view
Manage Files: Frontend design to delete file in grid view
Manage Files: Implement validation of 5mb on upload
Manage Files: Implement searching with date and Name

vikasranatalentelgia committed:
CAN-426, 427, 430, 431

Manage Files: Frontend design to delete files in list view
Manage Files: Frontend design to delete file in grid view
Manage Files: Implement validation of 5mb on upload
Manage Files: Implement searching with date and Name

akuks commented on src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx:
```diff
@@ -99,60 +113,134 @@ const UploadFileUI = ({
   const uploadOptionsShow = () => dispatch(showUploadOptions());
   const hideButtonAdd = () => dispatch(hideAddButton());
   const shownAddButton = () => dispatch(showAddButton());
   const StatushideFile = () => dispatch(hideFileStatus());
```
Please change it to camelcase

vikasranatalentelgia replied:
Done

akuks commented on src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx:
```diff
@@ -99,60 +113,134 @@ const UploadFileUI = ({
       </Menu.Item>
       <Menu.Item>
-        <a>
+        <a
```
Please use `Link` on all the anchor links

vikasranatalentelgia replied:
changed to div

akuks commented on src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx:
```diff
@@ -99,60 +113,134 @@ const UploadFileUI = ({
       </Menu.Item>
       <Menu.Item>
-        <a>
+        <a
```
Please use `Link` on anchorlinks

vikasranatalentelgia replied:
changed to div

akuks commented on src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx:
```diff
@@ -335,13 +456,39 @@ const UploadFileUI = ({
+                    //   removeFiles(originNode, file, currFileList,)
+                    // }
+                    />
+                    <img src={file.thumbUrl} height="150px" width="140px" />
```
use Image for images

vikasranatalentelgia replied:
done

akuks commented on src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx:
```diff
@@ -361,7 +508,7 @@ const UploadFileUI = ({
                       <div className={styles.dropdown}>
-                        <Dropdown overlay={menu(i)} trigger={["click"]}>
+                        <Dropdown overlay={menu(i, item)} trigger={["click"]}>
                           <a
```
Please use `Link` on anchorlinks

vikasranatalentelgia replied:
done

akuks commented on src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx:
```diff
@@ -376,7 +523,7 @@ const UploadFileUI = ({
                   <div className={styles.dropdown_menu}>
-                    <Dropdown overlay={menu_files(i)} trigger={["click"]}>
+                    <Dropdown overlay={menu_files(i, item)} trigger={["click"]}>
                       <a
```
Please use `Link` on anchorlinks

vikasranatalentelgia replied:
done

akuks commented on src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx:
```diff
@@ -670,15 +827,15 @@ const UploadFileUI = ({
         width={400}
         closeIcon={<CloseCircleOutlined />}
       >
         <Form
```
Please move Form in a separate component

vikasranatalentelgia replied:
done

vikasranatalentelgia committed:
Merge branch 'development' into CAN-115

vikasranatalentelgia committed:
CAN-429, 428, Code refactor

Manage Files: Frontend implementation to Open Folder, Edit Folder Name and delete folder in grid view
Manage Files: Frontend implementation to Open Folder, Edit Folder Name and delete folder in list view

vikasranatalentelgia committed:
Code refactor

Code refactor
